code/modules/research/nanites/nanite_programs/utility.dm

```diff
@@ -198,6 +198,75 @@
 /datum/nanite_program/metabolic_synthesis/active_effect()
 	host_mob.nutrition -= 0.5
 
+/datum/nanite_program/research
+	name = "Distributed Computing"
+	desc = "The nanites aid the research servers by performing a portion of its calculations, increasing research point generation."
+	use_rate = 0.2
+	rogue_types = list(/datum/nanite_program/toxic)
+
+/datum/nanite_program/research/active_effect()
+	if(!iscarbon(host_mob))
+		return
+	var/points = 1
+	if(!host_mob.client) //less brainpower
+		points *= 0.25
+	SSresearch.science_tech.add_point_list(list(TECHWEB_POINT_TYPE_GENERIC = points))
+	
+/datum/nanite_program/bitcoin
+	name = "Cryptocurrency Processing"
+	desc = "The nanites automatically mine cryptocurrency, automatically transferring the resulting credits to the station's account."
+	use_rate = 0.3
+	rogue_types = list(/datum/nanite_program/toxic)
+
+/datum/nanite_program/bitcoin/active_effect()
+	if(!iscarbon(host_mob))
+		return
+	var/points = 4 // 240/min
+	if(!host_mob.client)
+		points *= 0.25
+	SSshuttle.points += points
+	
+/datum/nanite_program/triggered/researchplus
+	name = "Neural Network"
+	desc = "The nanites link the host's brains together forming a neural research network, that becomes more efficient with the amount of total hosts. \
+			Can be overclocked (by triggering) to increase research output at the cost of increased nanite consumption and rapid brain decay from the host."
+	use_rate = 0.4
+	rogue_types = list(/datum/nanite_program/brain_decay)
+	var/overclock = FALSE
+
+/datum/nanite_program/triggered/researchplus/enable_passive_effect()
+	. = ..()
+	if(!iscarbon(host_mob))
+		return
+	SSnanites.neural_network_count++
+
+/datum/nanite_program/triggered/researchplus/disable_passive_effect()
+	. = ..()
+	if(!iscarbon(host_mob))
+		return
+	SSnanites.neural_network_count--
+	
+/datum/nanite_program/triggered/researchplus/active_effect()
+	if(!iscarbon(host_mob))
+		return
+	var/mob/living/carbon/C = host_mob
+	var/points = round(SSnanites.neural_network_count / 15, 1)
```

Comment:
Doesn't this mean zero yield until you have 8 participants?

Reply:
Good point, i'll make it is smaller increments with a minimum

Reply:
Also 1 brainloss means 60 loss per minute, doesn't it?
It seems a little steep since that's 100 seconds to major brain damage and 200 seconds to death.

Reply:
Considering other lethal programs it's pretty much on par. You have more than three minutes to find mannitol, shock/emp yourself, or change the program. I should add a warning though.